pkg/geo/wkt/lex.go

```diff
@@ -36,21 +36,28 @@ type ParseError struct {
 	problem string
 	pos     int
 	str     string
+	hint    string
 }
 
 func (e *ParseError) Error() string {
-	return fmt.Sprintf("%s at pos %d\n%s\n%s^", e.problem, e.pos, e.str, strings.Repeat(" ", e.pos))
+	// TODO(ayang) rethink position marker since input can be multiple lines, maybe use a "<-"?
```

Comment:
i would suggest only showing the line which is relevant, and maybe truncating output if the line is too long, e.g.

`...EOMETRYCOLLECTION (n`

a separate PR is acceptable if desired.

Reply:
Ok will do in a separate PR, will need to make the lexer track line numbers too.